pimd/pim_iface.c

```diff
@@ -1241,7 +1242,9 @@ static struct gm_join *igmp_join_new(struct interface *ifp,
 
 	return ij;
 }
+#endif
```

Comment:
If the `#if`...`#endif` block is longer than maybe 10-15 lines, please add a comment on the `#endif` that says what the condition on the `#if` was; the code becomes a bit hard to read without it.

Reply:
Fixed